src/client/src/orangehrmAdminPlugin/pages/systemUser/EditSystemUser.vue

```diff
@@ -225,29 +233,6 @@ export default {
           this.onCancel();
         });
     },
-    validateUserName(user) {
```

Comment:
I feel better to revert this and reuse `/api/v2/admin/validation/user-name`, and will remove this entity from the new API

Reply:
reverted